README.md

```diff
@@ -20,6 +20,7 @@ Linux amd64 Tags
 | `4-slim`                                 | Debian 11  |
 | `4-appservice` | Debian 11  |
 | `4-dotnet6-core-tools` |  Debian 11  |
+| `4-appservice-quickstart` | Debian 11 |
```

Comment:
In my opinion we should consider not listing the quickstart images as they are equivalent to the -appservice images.  We don't want customers to arbitrarily pin to the quickstart images thinking they are getting some benefit from it. Rather have all customers targetting the same appservice images. Makes hotfixes, future changes easier.

Reply:
Resolving the comment and merging the PR. @Cooperlink - lets sync up offline if you have more concerns